src/google-calendar-api.js

```diff
@@ -3,28 +3,38 @@ var google = require('googleapis');
 var calendar = google.calendar('v3').events;
 var moment = require('moment');
 
-var credentials = new Promise((resolve, reject) => {
-	const authFactory = new GoogleAuth();
-	const jwtClient = new authFactory.JWT(
-		process.env.GOOGLE_CLIENT_EMAIL,
-		null,
-		process.env.GOOGLE_PRIVATE_KEY,
-		['https://www.googleapis.com/auth/calendar']
-	);
+var credentials = (function() {
```

Comment:
I think that might be a good idea if we wanted to reference the Google API in new classes. What's your reasoning for moving the block to its own module?